src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color,double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: "
+                + getArea() + " sq. units, side: "
+                + side + " units, color: "
+                + color);
```

Comment:
Use 'getColor()' to access the 'color' property inherited from the 'Figure' class, as direct access to 'color' may not work if the 'color' field in the superclass is private and lacks a getter method.